firedantic/tests/tests_async/test_model.py

```diff
@@ -95,6 +95,52 @@ async def test_find(configure_db, create_company, create_product):
         await Product.find({"product_id": {"<>": "a"}})
 
 
+@pytest.mark.asyncio
+async def test_find_not_in(configure_db, create_company):
+    ids = ["1234555-1", "1234567-8", "2131232-4", "4124432-4"]
+    companies = []
+    for company_id in ids:
+        companies.append(await create_company(company_id=company_id))
+
+    found: List[Company] = await Company.find(
```

Comment:
Is it really necessary to add explicit type annotations there? it makes the code a bit less readable, but at least my IDE handles it perfectly: 
<img width="637" alt="image" src="https://user-images.githubusercontent.com/9670990/119097329-433fba80-ba1d-11eb-9c30-d507e85dc8d2.png">

Reply:
Fixed as well; a bit too much of copy & paste apparently.